docs/01-infrastructure-azure.md

```diff
@@ -236,7 +236,7 @@ Create VM
 azure vm create \
 	--resource-group the-hard-way \
 	--name etcd0 \
-	--vm-size Standard_D4 \
```

Comment:
I'd go with something bigger than an A1.  In order to move disks, at least 2 data disk spots are needed -- ideally more than that because at least one fix reserves a spot.